roundtripper/roundtripper.go

```diff
@@ -44,20 +44,23 @@ const (
 	BucketURL               = "BLOB_URL"
 )
 
+var counter int64
+
 // RateLimitRoundTripper is a rate-limit aware http.Transport for Github.
 type RateLimitRoundTripper struct {
 	Logger         *zap.SugaredLogger
 	InnerTransport http.RoundTripper
 }
 
 type RoundRobinTokenSource struct {
-	counter      int64
 	AccessTokens []string
+	log          *zap.SugaredLogger
 }
 
 func (r *RoundRobinTokenSource) Token() (*oauth2.Token, error) {
-	c := atomic.AddInt64(&r.counter, 1)
+	c := atomic.AddInt64(&counter, 1)
 	index := c % int64(len(r.AccessTokens))
+	r.log.Infof("using token %d and the len of the tokens are %d and in the c is %d", index, len(r.AccessTokens), c)
```

Comment:
We don't need to log c
just index and len is good
can you put len(r.AccessTokens) in a local
also, message can be shortened
using token %d of total %d.

Reply:
Agreed! Thanks